tiledb/api/src/context.rs

```diff
@@ -72,8 +87,9 @@ where
     }
 }
 
+#[derive(Clone)]
 pub struct Context {
-    raw: RawContext,
+    raw: Rc<RefCell<RawContext>>,
```

Comment:
Why do you need the RefCell?

Reply:
I probably don't in hindsight. I just did `Rc<RefCell<T>>` because that's basically used in all the places. I'll try and remove it and see if we can get away without it.

Reply:
I'm sure you can.  A Rust-native thing would need it for interior mutability, but we get that via the `ffi`

Reply:
Oh, I'm fairly certain of it as well. Just not 100% certain that there's nothing subtle relying on a trait for RefCell that's not apparent. Easiest way to find out is to just give it a whirl which should be something like two or three lines of code.